tool/generate_example.dart

```diff
@@ -56,6 +57,7 @@ void main(List<String> arguments) {
 
   List<String> generatedOutput = [
     '',
+    "import 'package:flutter/material.dart';",
```

Comment:
Not sure why we need to import `flutter/material.dart`

Reply:
This is required for `Colors` when Duotone icons are used - perhaps we could comment this out, then  add a section to the readme that it needs to be uncommented for Pro or Duotone implementations (which are currently only a Pro feature)?  This gives the best of both worlds - it's readily available, but doesn't affect the package size for `free` users.  We could add it alongside the instructions for uncommenting the `pubspec.yaml`.

Reply:
Scratch this - it isn't necessary here, and I have removed it.